lib/api/core/plugins/pluginContext.js

```diff
@@ -27,21 +47,20 @@ class PluginContext {
     // we have a circular dependency between Kuzzle and the plugins.
     // We cannot get Kuzzle constructor from the global scope
     const Kuzzle = require('../../kuzzle');
-    let pluginInternalEngine;
 
     this.accessors = {};
     this.config = kuzzle.config;
     this.constructors = {
-      Dsl: require('../../dsl'),
-      Request: instantiateRequest,
       RequestContext,
       RequestInput,
+      Dsl: require('../../dsl'),
+      Request: instantiateRequest,
       BaseValidationType: require('../validation/baseType')
     };
     this.errors = require('kuzzle-common-objects').errors;
 
     if (kuzzle instanceof Kuzzle) {
-      pluginInternalEngine = new InternalEngine(kuzzle, `%${pluginName}`);
+      const pluginInternalEngine = new InternalEngine(kuzzle, `%${pluginName}`);
       pluginInternalEngine.init(new PluginInternalEngineBootstrap(kuzzle, pluginInternalEngine));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 97 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)